### PR TITLE
Shear strain corrections

### PR DESCRIPTION
**Pull request by bhalonen**
__Related PRs__
- https://github.com/relinc/ncorr_cpp/pull/2

Able to use read any strain average as a DIC map.

### CODE REVIEW

File: Libraries/src/net/relinc/libraries/application/Bar.java [hunk 1]
```diff
@@ -16,7 +16,7 @@
 	public double diameter;
 	public double speedLimit;
 	public double yield;
-	private double poissonsRatio;
+	private double poissonsRatio=.33;
```
Comment (markhalonen) [inline Libraries/src/net/relinc/libraries/application/Bar.java]: This a speculative code change for safety or was this a correction of incorrect behavior? Not opposed to the default value, just curious

**Comment by markhalonen** [comment]
Shouldn't have commits from other PR... and looks like this needs to be run alongside changes from https://github.com/relinc/ncorr_cpp/pull/2

File: Libraries/src/net/relinc/libraries/application/Bar.java (hunk 1 above)
Comment (bhalonen) [reply Libraries/src/net/relinc/libraries/application/Bar.java]: I think it came out to zero once when I was using it on old bar set-ups.

Comment (bhalonen) [reply Libraries/src/net/relinc/libraries/application/Bar.java]: It is a default, but generally close.

**Pull request closed without merging**